ui/src/pages/frontsite/components/Series/SeriesAbstract.js

```diff
@@ -0,0 +1,23 @@
+import ShowMore from 'react-show-more';
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+
+export class SeriesAbstract extends Component {
```

Comment:
Actually this could be general component `Abstract`, both for series and documents, as I remember they are the same. they work also the same way

Reply:
Yes, they do. It would make sense indeed to use the same component for those two.